CONTRIBUTING.md

```diff
@@ -64,3 +64,55 @@ Other Tips
     your existing commit message by running:
 
         $ git commit --amend
+
+Checklist for Contributed Code
+------------------------------
+
+A pull request for a new feature will be reviewed to see if it meets the following requirements.  For any pull request, an astropy maintainer can help to make sure that the pull request meets the requirements for inclusion in the package.  
+
+**Scientific Quality**
```

Comment:
@embray - as I was the one suggesting to put it higher up...: my sense was that the checklist should be top-level down, so that one doesn't end up being worried about typos and PEP8 issues before having thought through the code well.  I do agree that typical submissions are fine in this regard, but I think that doesn't change that it should be the first think to check (if it is easy, so much the better, move on!).